sample.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>I am normal</p>
```

Comment:
Let's remove this file, if not needed.
Also, might be good to add `*.html` files in the .gitignore.

Reply:
I intentionally added this sample file for anyone to check if its their HTML which has some issue or this project.
Since sometimes a broken HTML would not render and it will be a little time consuming for someone to verify what's broken, is it the HTML or this code.
Am happy to probably move it to a separate sample or docs folder if you think it's a better idea or remove it altogether.